neutromeratio/constants.py

```diff
@@ -23,6 +23,10 @@
 # ANI-1 units and conversion factors
 ani_distance_unit = unit.angstrom
 hartree_to_kJ_mol = 2625.499638
+kJ_mol_to_kT = 0.40090737504650614
```

Comment:
warning: won't update if `temperature` is changed

Reply:
that's a good point -- I will change this